phpunit.xml.dist

```diff
@@ -2,11 +2,8 @@
 <phpunit bootstrap="./tests/bootstrap.php" colors="true">
 
     <testsuites>
-        <testsuite name="unit">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Unit</directory>
-        </testsuite>
-        <testsuite name="functional">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Functional</directory>
+        <testsuite name="components">
```

Comment:
as far as i see only the `Hash` component is not structured like that! it's ok for me to merge it but we should keep this structure although and move this tests into the Unit folder.

Only to say for bigger refactorings its interresting to only run the functional testcases to see if the function is ok and run the unit tests later.